### ISSUE TITLE
APK Download

### ISSUE THREAD

**Issue by IzzySoft** [scheibler/WalkersGuide-Android#1]
First thanks a lot for making your app free, libre & foss! I would like to make it available for the FOSS world via [my F-Droid repo](https://apt.izzysoft.de/fdroid), also to make it easier to discover (and keep it updated) by those **outside** of Google's walled garden.

I see you offer the APKs for download on your website. Unfortunately, my updater cannot automatically fetch from there as it cannot know the names in advance. It would however work fine were the APKs attached to their corresponding [releases](https://github.com/scheibler/WalkersGuide-Android/releases) here. That would enable it to fetch updates within 24h of your making them available here.

Would you consider doing that? Thanks in advance!

**Comment by scheibler** [comment]
Of course, please feel free to add WalkersGuide to your fDroid repository.

I've created a release on GitHub. Please reopen this issue, if that doesn't work for you.

**Comment by IzzySoft** [comment]
Thanks a lot! I've already [added your app](https://apt.izzysoft.de/fdroid/index/apk/org.walkersguide.android) yesterday setting updates to manual, as I felt it important enough to make this exception (there never can be enough apps to help those having lost or otherwise missing a sense we take for granted). The new release is perfect, I've adjusted the config here and enabled automated updates (to avoid the huge delay manual ones cause). Be welcome to [pick a badge](https://gitlab.com/IzzyOnDroid/repo/#if-my-app-is-available-here-do-you-have-a-badge-i-can-use-to-link-to-it) if you want to link there, e.g. from the Readme. And I gladly provide you the metadata I've set up, using Fastlane format (see my [Fastlane Cheat Sheet](https://gitlab.com/-/snippets/1895688) for details) if you want to be in control of how your app is presented here (Fastlane would then be pulled along with each new release).

That said, I've also added [Tactile Clock](https://apt.izzysoft.de/fdroid/index/apk/de.eric_scheibler.tactileclock). Do you plan updates for that one, too? Or is it rather "mature" and "feature-complete", so further updates are not to be expected?

**Comment by scheibler** [comment]
Thank you for adding WalkersGuide.

And yes, Tactile Clock is unlikely to receive updates in the future. I'm still using it with Android
13 and I'm happy with its functionality. I would only release a new version, if I'd encounter
problems with new Android versions.

In fact, it's nice to see it on fDroid, cause Google removed it from the play store for new users
recently. Turns out, that the app is too old for Google, although it still works fine, at least for now.

**Comment by IzzySoft** [comment]
> I would only release a new version, if I'd encounter problems with new Android versions.

I've set it for "manual updates", so I check only every now and then (usually about 4 times a year).

> In fact, it's nice to see it on fDroid, cause Google removed it from the play store for new users recently.

It's only in my repo, not on F-Droid.org – but if you wish it could be tried to include it there (both apps in fact seem to meet the inclusion criteria, from what I can see). Getting that process started is as easy as [opening an RFP](https://gitlab.com/fdroid/rfp/-/issues/). As I've set it up here already, I can provide most of the metadata for that – especially the Fastlane structures needed (see my [Fastlane Cheat Sheet](https://gitlab.com/-/snippets/1895688)), which I could provide via PR here if you wish.

"Oldies but goodies": if old==useless, we had no good wines and might be considered obsolete ourselves. I've got to do some weeding in my repo, too, for apps having received no updates for a long time – but I do not remove "just by age". Some apps are simply mature & feature-complete :wink: And still very useful for some. Like your apps.

> Thank you for adding WalkersGuide.

Gladly! WalkersGuide received some praise within 24h of being added I feel the need to share with you:

> Man, the [#walkersGuide](https://floss.social/tags/walkersGuide) is such a nice, amazing, powerfull [#OSM](https://floss.social/tags/OSM) app. My initial impressions are life changing. I need to explore it better of course. Huge thanks!
